Cartfile.private

```diff
@@ -1,2 +1,2 @@
-github "Quick/Nimble"  ~> 5.1
-github "Quick/Quick" ~> 0.10
+github "Quick/Nimble"
```

Comment:
Could we adjust target version instead of getting whatever is on master? I know that this don't matter for clients of the library, but I still believe that constraining to major is useful.